src/c/http.c

```diff
@@ -413,8 +425,8 @@ int main(int argc, char *argv[]) {
     return 1;
   }
 
```

Comment:
Consider adding another `setsockopt` that disables `IPV6_V6ONLY` to explicitly allow IPv4-to-IPv6 mapped addresses. Not all operating systems use the same default value.